_includes/pbc/all/install-features/202403.0/marketplace/install-the-marketplace-merchant-feature.md

```diff
@@ -0,0 +1,1653 @@
+| SpyMerchantSearchEntity                    | class    | Created | src/Generated/Shared/Transfer/SpyMerchantSearchEntityTransfer                    |
+| SpyMerchantStorageEntity                   | class    | Created | src/Generated/Shared/Transfer/SpyMerchantStorageEntityTransfer                   |
+| SpyMerchantUserEntity                      | class    | Created | src/Generated/Shared/Transfer/SpyMerchantUserEntityTransfer                      |
+| UrlStorage.fkResourceMerchant              | property | Created | src/Generated/Shared/Transfer/UrlStorageTransfer                                 |
+
+{% endinfo_block %}
+
+## 3) Set up configuration
+
+{% info_block warningBox "Warning" %}
+
+Apply the following changes only if you have the [Glue Authentication](/docs/scos/dev/migration-concepts/migrate-to-decoupled-glue-infrastructure/decoupled-glue-infrastructure-integrate-the-authentication.html) feature installed and if you would like to specify a list of endpoints access to which is allowed for merchant users.
+
+{% endinfo_block %}
+
+**src/Pyz/Zed/OauthMerchantUser/OauthMerchantUserConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\OauthMerchantUser;
+
+use Spryker\Zed\OauthMerchantUser\OauthMerchantUserConfig as SprykerOauthMerchantUserConfig;
+
+class OauthMerchantUserConfig extends SprykerOauthMerchantUserConfig
+{
+    /**
+     * Specification:
+     * - Returns a list of configurations for endpoints accessible to merchant users.
+     * - Structure example:
+     * [
+     *      '/example' => [
+     *          'isRegularExpression' => false,
+     *      ],
+     *      '/\/example\/.+/' => [
+     *          'isRegularExpression' => true,
+     *          'methods' => [
+     *              'patch',
+     *              'delete',
+     *          ],
+     *      ],
+     * ]
+     *
+     * @api
+     *
+     * @return array<string, mixed>
+     */
+    public function getAllowedForMerchantUserPaths(): array
+    {
+        return [];
+    }
+}
+```
+
+### 4) Add Zed translations
+
+Generate new translation cache for Zed:
+
+```bash
+console translator:generate-cache
+```
+
+### 5) Set up behavior
+
+Enable the following behaviors by registering the plugins:
+
+| PLUGIN                                               | DESCRIPTION                                                                                              | PREREQUISITES | NAMESPACE                                                            |
+|------------------------------------------------------|----------------------------------------------------------------------------------------------------------|---------------|----------------------------------------------------------------------|
+| MerchantProfileExpanderPlugin                        | Expands merchant with profile data.                                                                      |               | Spryker\Zed\MerchantProfile\Communication\Plugin\Merchant            |
```

Comment:
* Deprecated one, has to be substituted by the \Spryker\Zed\MerchantProfile\Communication\Plugin\Merchant\MerchantProfileMerchantBulkExpanderPlugin.